### DISCUSSION TITLE
is hardware transcoding working ?

### DISCUSSION THREAD

**Discussion by alez007** [immich-app/immich#1578] [Q&A] [answered]
Hey, wondering if hardware transcoding is working, I saw a merged PR but no documentation as to how to make it work.

Thanks

**Comment by bo0tzz** [comment, marked as answer]
Which PR are you referring to? Hardware transcoding has not been implemented yet.

**Reply by alez007** [reply]
thanks, I think might have been confused by the merged PR from this issue: https://github.com/immich-app/immich/issues/203
